### DISCUSSION TITLE
When upload document to RAGFlow, '_xOOOD_' was found after parsing.

### DISCUSSION THREAD

**Discussion by Zhengbiao-Xue** [infiniflow/ragflow#2734] [Q&A]
RAGFlow使用Q&A解析格式，上传文档，解析文档，查看RAGFlow后台知识库,发现有_xOOOD_字符
![_xOOOD_字符](https://github.com/user-attachments/assets/d6673948-f0f0-4cf3-b652-951919f78991)
我的邮件是：2573223949@qq.com，方便联系我

**Comment by KevinHuSh** [comment]
RAGFlow has no chance to add these charactors.
